hr_employee_age/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2015 Salton Massally (<smassally@idtlabs.sl>).
```

Comment:
It should be w/o `(C)` as per https://github.com/OCA/maintainer-tools/blob/1ae9b316377defbde06b67e08763f574260ee153/template/module/__manifest__.py#L2